rules/validator.go

```diff
@@ -77,8 +77,16 @@ func (v *validator) validateMappingRules(mrv map[string]*MappingRuleView) error
 			return err
 		}
 
+		// Validate the metric types.
+		types, err := v.opts.MetricTypesFn()(view.Filters)
+		if err != nil {
+			return err
+		}
+		if len(types) == 0 {
+			return fmt.Errorf("mapping rule %s does not match any allowed metric types", view.Name)
```

Comment:
nit: Include the type that the mapping rule has? To help debug typos

Reply:
Sure, can include the filter